README.md

```diff
@@ -67,6 +67,13 @@ If you want correct CPU temperature displayed in `byobu` status notifications, a
 
     MONITORED_TEMP=/sys/class/hwmon/hwmon1/temp1_input
 
+## Display
+
+### Fractional Scaling
+On KDE, Deepin and other Qt-based desktop environments, fractional scaling is supported without any hassle but this is not the case for GTK-based desktop environments such as GNOME and XFCE. Using Wayland can be an option, but the apps that don't support Wayland natively will still have issues such fuzziness all over the window. A workaround for Xorg allows for 150% scaling without any tear at the const of a little performance drop:
+Copy `20-intel.conf` and `10-monitor.conf` to `/etc/X11/xorg.conf.d/` and reboot.
+It is worth noticing that kernels between 5.0 and 5.5 have known issues with desktop scaling, fixed in 5.5. Ubuntu users can use Ubuntu-18.04-HWE kernel that contains the fix and also is free of the log flooding issue. For other than Debian-derived distribution users, they should compile their kernel with the [following](#touchpad) patch.
```

Comment:
"log flooding issue" should be a link, since this issue is discussed a couple of sections below, and and a first-time reader may not have an idea about it.

Reply:
Also, the last sentence is not needed, IMO. Touchpad issues belong in its own section, and some of the distros already have kernels 5.2+ that don't need the patch, others don't, and I think it's basically up to the user to grok which parts of the guide apply to the distro they're using, if that doesn't happen to be Debian ;)

Reply:
Changed.